### DISCUSSION TITLE
Colorblind friendly color palette

### DISCUSSION THREAD

**Discussion by harish5p** [theOehrly/Fast-F1#398] [Ideas]
I use these colors which are tested for colorblindness and verified with several folks as well, these are effective and everyone can distinguish these colors from each other. If it's possible implement these

This is how the colors look on a dark background:
![image](https://github.com/theOehrly/Fast-F1/assets/20576094/d9a08733-1307-421f-bddf-a4ebd3945cfe)

This is how the colors look on a light background:
![image](https://github.com/theOehrly/Fast-F1/assets/20576094/21579666-82de-47a4-8fa7-c52107617146)

Colors: https://gist.github.com/TracingInsights/53bb75a12d232d7a2e5293b6ac886d44

F1 Api has different team names to Ergast. Both are included in the code

If you would like to simulate colorblindness, this tool helps with that: https://colororacle.org

**Comment by Casper-Guo** [comment]
This is an amazing idea. You should 100% PR

**Comment by theOehrly** [comment]
This is really cool, accessibilty is very important. 
The whole plotting stuff in FastF1 needs to be rewritten anyway. Support for different colormaps should then be added so this can be added. I'm planning to do this "soon" (hopefully actually soon).
